src/api/app/controllers/webui/patchinfo_controller.rb

```diff
@@ -25,10 +25,11 @@ def create
     redirect_to edit_patchinfo_path(project: @project, package: @package)
   end
 
-  def updatepatchinfo
+  def fetch_issues
     authorize @project, :update?
 
-    Patchinfo.new.cmd_update_patchinfo(params[:project], params[:package])
+    message = 'updated via fetch_issues call'
+    Patchinfo.new.cmd_update_patchinfo(params[:project], params[:package], message)
```

Comment:
:thinking: I would replace the `message` variable with the actual message. Given that it's only used once.

Reply:
done, thanks!